seed/api/v1/urls.py

```diff
@@ -5,27 +5,28 @@
 :author
 """
 
-from django.conf.urls import url
+from django.conf.urls import url, include
+from rest_framework import routers
 
+from seed.api.v1.views import ColumnViewSetV1
 from seed.views.main import (
     search_buildings,
-    get_default_columns,
     set_default_columns,
     get_default_building_detail_columns,
     set_default_building_detail_columns,
-    get_columns,
-    delete_file,
     public_search,
     export_buildings_download,
     delete_organization_inventory,
 )
 
-# prefix, to revert back to original endpoints, leave this blank
+router = routers.DefaultRouter()
+router.register(r'columns', ColumnViewSetV1, base_name="columns")
 
 urlpatterns = [
     # template routes
+    url(r'^', include(router.urls)),
+
     url(r'^search_buildings/$', search_buildings, name='search_buildings'),
-    url(r'^get_default_columns/$', get_default_columns, name='get_default_columns'),
     url(r'^set_default_columns/$', set_default_columns, name='set_default_columns'),
```

Comment:
Was there a reason this `set` method wasn't pulled into the viewset?  Should this be added to a followup todo list?

Reply:
yeah, it should be added to a todo list. The set_ and get_ here are never used as far as I can tell.